app/upgrades/v6.0.0/upgrades.go

```diff
@@ -170,9 +173,35 @@ func SetupRewards(ctx context.Context, bk bankkeeper.Keeper) error {
 	return nil
 }
 
+func SetupDenomMetadata(ctx context.Context, bk bankkeeper.Keeper) {
+	bk.SetDenomMetaData(ctx, banktypes.Metadata{
```

Comment:
values are set based on my inspiration by the way other chains do that. see
```
gaiad q bank denom-metadata uatom --node https://cosmoshub.tendermintrpc.lava.build:443
gaiad q bank denom-metadata uosmo --node https://rpc.osmosis.zone
gaiad q bank denom-metadata utia --node https://rpc.lavenderfive.com:443/celestia
```

the other way could be setting DenomUnits similarly to how celestia did:
```
"denom_units": [
      {
        "denom": "utia",
        "aliases": [
          "microtia"
        ]
      },
      {
        "denom": "TIA",
        "exponent": 6
      }
    ]
```
so instead of searching for the right unit by iterating over aliases, we could just rely on the denom. that sounds better for me, but I'm not sure what would be the right way here

Reply:
we decided to keep it as is